docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Database Access with Cassandra and ScyllaDB
+description: How to configure Teleport Database Access with Azure Database for Cassandra and ScyllaDB.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="11.0"
+>
+  Database access for Cassandra & ScyllaDB is available starting from Teleport `v11.0`.
+</Details>
+
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up Teleport to access your self-hosted Cassandra or ScyllaDB.
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redis Self-Hosted](../../../img/database-access/guides/cassandra_keyspaces_slefhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redis Cloud](../../../img/database-access/guides/cassandra_keyspaces_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- Self-hosted Cassandra or ScyllaDB instance.
+- cqlsh cassandra client installed and added to your system's `PATH` environment variable.
+- - A host, e.g., an Amazon EC2 instance, where you will run the Teleport Database
+Service.
+
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+```code
+$teleport db start \
```

Comment:
Question: Is this the default manner of starting the database service? I initially thought it was the configuration file approach with `teleport db configure create ...` and `teleport start -c`.

Reply:
I don't think that we have a defautl way. 
For self hosted database we recumbent to use `teleport start db` approach where as for AWS related db services  if auto discovery is supported `teleport db configure` should be preferred. The AWS Keystores is quite different because you can have only it enabled or disable so autodiscovery flow doesn't fit here.